lib/network/RoundRobin.ts

```diff
@@ -55,8 +69,8 @@ class RoundRobin {
         this.pickCount = 0;
     }
 
-    _validateHostObj(hostItem) {
-        const hostItemObj = {};
+    _validateHostObj(hostItem: string | { host: string; port: string }): { host: string; port?: number } {
```

Comment:
Why not using an interface to avoid repeating this custom type everywhere?

Reply:
Well see, a declaration was useless